HACKING/launch-environment.sh

```diff
@@ -1,11 +1,13 @@
 #!/bin/bash
-IMAGINE_DIR=/home/$USER/Code/ImagineRIT2022
+IMAGINE_DIR=/home/thegreenone/Documents/imagine2022-embedded
```

Comment:
Please change this back. Did the `$USER` variable not work for you? That should be an environment variable.